atc/exec/load_var_step_test.go

```diff
@@ -362,6 +363,27 @@ var _ = Describe("LoadVarStep", func() {
 				Expect(stepErr).To(MatchError(ContainSubstring("failed to parse some-resource/a.yaml in format yaml")))
 			})
 		})
+
+		Context("when file path artifact is not registered", func() {
+			BeforeEach(func() {
+				loadVarPlan = &atc.LoadVarPlan{
+					Name: "some-var",
+					File: "some-resource-not-in-the-registry/a.json",
+				}
+
+				fakeArtifactStreamer.StreamFileFromArtifactReturns(&fakeReadCloser{str: plainString}, nil)
+			})
+
+			It("step should fail", func() {
+				Expect(stepErr).To(HaveOccurred())
+				Expect(stepErr).To(
+					Equal(artifact.UnknownArtifactSourceError{
+						Name: "some-resource-not-in-the-registry",
+						Path: "a.json",
+					}))
+				Expect(stepErr).To(MatchError("unknown artifact source: 'some-resource-not-in-the-registry' in file path 'a.json'"))
```

Comment:
previously would be `"unknown artifact source: 'some-resource-not-in-the-registry' in task config file path 'a.json'"`